### ISSUE TITLE
Missing attachto documentation

### ISSUE THREAD

**Issue by grimalschi** [paldepind/snabbdom#137]
I found attachto module in helpers folder, but there is no description. Only tests.

I offer to add description for attachto.

**Comment by paldepind** [comment]
Yes that would be great :+1:  It's useful for implementing modals and similar things.

**Comment by avesus** [comment]
I've found this in 2015 Gitter messages:

[attachTo](https://github.com/paldepind/snabbdom/blob/master/helpers/attachto.js) is a function that makes it possible to declare that a vnode should be attached to something else than it's parent.

It contains the low level functionality of [react-portal](https://github.com/tajo/react-portal).

Very simple [modal example](https://github.com/paldepind/noname-functional-frontend-framework/tree/master/examples/modal) demonstrates how to use it.
